### DISCUSSION TITLE
please help a new DiaNN user: can't find the report tsv or pdf

### DISCUSSION THREAD

**Discussion by rmeng2** [vdemichev/DiaNN#887] [Q&A] [answered]
I can't find any results after searching is finished and I see the following:
"DIA-NN exited"
......
"PDF report will be generated in the background"
Not in my designated folder, not coming up in my searching of the whole computer.
Please help??? thank you in advance!

**Comment by vdemichev** [comment]
How does the full log look like?

**Reply by rmeng2** [reply]
Thank you for replying. I am pasting it below. Does it look all normal to you?
===============================================
Current date and time: Thu Dec 14 10:19:17 2023
CPU: GenuineIntel Intel(R) Core(TM) i5-8365U CPU @ 1.60GHz
SIMD instructions: AVX AVX2 FMA SSE4.1 SSE4.2 
Logical CPU cores: 8
Thread number set to 4
Output will be filtered at 0.01 FDR
Library-free search enabled
Min fragment m/z set to 100
Max fragment m/z set to 1500
N-terminal methionine excision enabled
In silico digest will involve cuts at K*,R*
Maximum number of missed cleavages set to 1
Min peptide length set to 7
Max peptide length set to 30
Min precursor m/z set to 300
Max precursor m/z set to 1500
Min precursor charge set to 2
Max precursor charge set to 3
Cysteine carbamidomethylation enabled as a fixed modification
Maximum number of variable modifications set to 2
Modification UniMod:35 with mass delta 15.9949 at M will be considered as variable
Modification UniMod:21 with mass delta 79.9663 at STY will be considered as variable
A spectral library will be created from the DIA runs and used to reanalyse them; .quant files will only be saved to disk during the first step
When generating a spectral library, in silico predicted spectra will be retained if deemed more reliable than experimental ones
Implicit protein grouping: protein names; this determines which peptides are considered 'proteotypic' and thus affects protein FDR calculation
Fixed-width center of each elution peak will be used for quantification
Interference removal from fragment elution curves disabled
Mass accuracy will be fixed to 5e-08 (MS2) and 2e-05 (MS1)
Exclusion of fragments shared between heavy and light peptides from quantification is not supported in FASTA digest mode - disabled; to enable, generate an in silico predicted spectral library and analyse with this library
The following variable modifications will be scored: UniMod:21 
WARNING: it's strongly recommended to use deep learning spectra/RTs prediction. If impossible because some unsupported modifications need to be searched, consider using either (i) the --strip-unknown-mods command or (ii) a 'training library' specified with the --learn-lib command. A training library might facilitate about 1.5x more IDs, deep learning - about 2x-3x more IDs.
WARNING: MBR turned off, two or more raw files are required

1 files will be processed
[0:00] Loading FASTA C:\Users\rmeng2\OneDrive - JNJ\Documents\_Analytes-Data\proteomics\SwissProt\uniprot_sprot.fasta\uniprot_sprot.fasta
[9:25] Processing FASTA

DIA-NN exited
DIA-NN-plotter.exe "C:\DIA-NN\1.8.1\report.stats.tsv" "C:\DIA-NN\1.8.1\report.tsv" "C:\DIA-NN\1.8.1\report.pdf"
PDF report will be generated in the background
======================================

**Reply by vdemichev** [reply, marked as answer]
Yes, nothing is happening because DIA-NN is not instructed to do anything.
For generating a spectral library in silico, need to keep the 'Deep learning' box checked (it gets checked automatically when FASTA digest is selected)

**Reply by rmeng2** [reply]
Oh thank you!
